applications/convection_diffusion_application/custom_utilities/bfecc_convection.h

```diff
@@ -266,58 +266,58 @@ template<std::size_t TDim> class BFECCConvection
     }
     
     
-    	void ResetBoundaryConditions(ModelPart& rModelPart, const Variable< double >& rVar) 
-		{
-				KRATOS_TRY
-				
-				ModelPart::NodesContainerType::iterator inodebegin = rModelPart.NodesBegin();
-				vector<unsigned int> node_partition;
-				#ifdef _OPENMP
-					int number_of_threads = omp_get_max_threads();
-				#else
-					int number_of_threads = 1;
-				#endif
-				OpenMPUtils::CreatePartition(number_of_threads, rModelPart.Nodes().size(), node_partition);
-				
-				#pragma omp parallel for
-				for(int kkk=0; kkk<number_of_threads; kkk++)
-				{
-					for(unsigned int ii=node_partition[kkk]; ii<node_partition[kkk+1]; ii++)
-					{
-							ModelPart::NodesContainerType::iterator inode = inodebegin+ii;
-
-							if (inode->IsFixed(rVar))
-							{
-								inode->FastGetSolutionStepValue(rVar)=inode->GetSolutionStepValue(rVar,1);
-							}
-					}
-				}
-			
-				KRATOS_CATCH("")
-		}
-		
-		void CopyScalarVarToPreviousTimeStep(ModelPart& rModelPart, const Variable< double >& rVar)
+        void ResetBoundaryConditions(ModelPart& rModelPart, const Variable< double >& rVar)
         {
-			KRATOS_TRY
-			ModelPart::NodesContainerType::iterator inodebegin = rModelPart.NodesBegin();
-			vector<unsigned int> node_partition;
-			#ifdef _OPENMP
-				int number_of_threads = omp_get_max_threads();
-			#else
-				int number_of_threads = 1;
-			#endif
-			OpenMPUtils::CreatePartition(number_of_threads, rModelPart.Nodes().size(), node_partition);
-			
-			#pragma omp parallel for
-			for(int kkk=0; kkk<number_of_threads; kkk++)
-			{
-				for(unsigned int ii=node_partition[kkk]; ii<node_partition[kkk+1]; ii++)
-				{
-					ModelPart::NodesContainerType::iterator inode = inodebegin+ii;
-				    inode->GetSolutionStepValue(rVar,1) = inode->FastGetSolutionStepValue(rVar);
-				}
-			}
-			KRATOS_CATCH("")
+                KRATOS_TRY
+
+                ModelPart::NodesContainerType::iterator inodebegin = rModelPart.NodesBegin();
+                vector<unsigned int> node_partition;
+                #ifdef _OPENMP
+                    int number_of_threads = omp_get_max_threads();
```

Comment:
ok, however here it would have been best to use omp utils

Reply:
It was already there, I just changed tabulation